src/amo/components/AddonTitle/index.js

```diff
@@ -14,6 +14,7 @@ import type { I18nType } from 'core/types/i18n';
 import './styles.scss';
 
 type Props = {|
+  ComponentTag?: string,
```

Comment:
react-bootstrap uses `as` as prop, and i find it nice, but maybe `component` would be enough.

Reply:
hmm..I thought ComponentTag was more cleared since it's not an actual component and just the tag name. But `as` looks interesting...
I'll take a look at that:)